housecanary/excel/utilities.py

```diff
@@ -56,8 +56,26 @@ def get_addresses_from_input_file(input_file_name):
     """Read addresses from input file into list of tuples."""
     with open(input_file_name, 'rb') as input_file:
         reader = csv.reader(input_file, delimiter=',', quotechar='"')
+
         addresses = map(tuple, reader)
-        return addresses
+
+        if len(addresses) == 0:
+            raise Exception('No addresses found in input file')
+
+        header_columns = list(column.lower() for column in addresses[0])
+
+        try:
+            address_index = header_columns.index('address')
+            zipcode_index = header_columns.index('zipcode')
+        except ValueError:
+            raise Exception("""The first row of the input CSV must be a header that contains \
+a column labeled 'address' and a column labeled 'zipcode'.""")
+
+        # skip the header row
+        if len(addresses) > 0:
+            addresses.pop(0)
+
+        return list((row[address_index], row[zipcode_index]) for row in addresses)
```

Comment:
You can remove the extra if statement above this if you change this to
```py
return [(row[address_index], row[zipcode_index]) for row in addresses[1:]]
```

Alternatively, you could do `header_columns = list(column.lower() for column in addresses.pop(0))` further up and skip the extra slice/pop here.